### PR TITLE
🚧 Modularize Bot Lists

### LINKED ISSUE #494: Modularize statistics posters

### PR DESCRIPTION
**Pull request by pedrofracassi**
Closes #494 

- [x] botlist.space
- [x] botsfordiscord.com
- [x] bots.ondiscord.xyz
- [x] discord.boats
- [x] discordboats.club
- [x] discordbotlist.com
- [ ] ls.terminal.ink
- [ ] discordbot.world
- [ ] discord.bots.gg
- [ ] discordsbestbots.xyz
- [ ] divinediscordbots.com
- [x] discordbots.org

### CODE REVIEW

**Comment by dpaiv0** [comment]
yo what the fuck this is actually awesome, this would look also very cool on [development visualization](https://gist.github.com/dpaiv0/9e7e08d0b09581db378b6b74b0ed2f0e)

File: src/botlists/DiscordBots.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    super()
+
+    this.name = 'discordbots.org'
+    this.envVars = ['DISCORDBOTS_TOKEN']
+  }
+
+  postStatistics ({ id, shardId, shardCount, serverCount }) {
+    return snekfetch
+      .post(`https://discordbots.org/api/bots/${id}/stats`)
+      .set('Authorization', process.env.DBL_TOKEN)
```
Comment (pedrofracassi) [inline src/botlists/DiscordBots.js]: whoopsie

**Comment by metehus** [comment]
@pedrofracassi hey

**Comment by pedrofracassi** [comment]
hmm

**Comment by pedrofracassi** [comment]
hmmmm

**Comment by MrJacz** [comment]
botcord.org when 😂

**Comment by MrJacz** [comment]
On a serious note. I would suggest using node-fetch instead of snekfetch. as snekfetch was deprecated. mention me on discord for a really nice fetch util

**Comment by pedrofracassi** [comment]
Oh yeah, this PR is stale and OOOOOOOOLD. I need to rewrite it using node-fetch.

**Comment by pedrofracassi** [comment]
Closing this as it will be replaced by [SwitchbladeBot/bot-list-poster](http://github.com/SwitchbladeBot/bot-list-poster).

**Pull request closed without merging**